pkg/policy/api/l4.go

```diff
@@ -150,6 +150,14 @@ type Listener struct {
 	// +kubebuilder:validation:MinLength=1
 	// +kubebuilder:validation:Required
 	Name string `json:"name"`
+
+	// Priority for this Listener that is used when multiple rules would apply different
+	// listeners to a policy map entry. Behavior of this is implementation dependent.
+	//
+	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:validation:Maximum=65535
```

Comment:
Even if it's implementation dependent, I think that starting with a smaller number provides better bounding of worst case behaviour. I'm a little concerned about having an API we can't express because of local fd limits.

```suggestion
	// +kubebuilder:validation:Maximum=100
```

Reply:
Sorry, earlier thought the limited number referred to the number of listeners in general! Totally makes sense to limit the range here to begin with.

Reply:
Done!